ra-tls/ra-tls-server/lib/Makefile

```diff
@@ -61,10 +74,14 @@ ifeq ($(HAVE_WOLFSSL_BENCHMARK), 1)
 	Wolfssl_C_Extra_Flags += -DHAVE_WOLFSSL_BENCHMARK
 endif
 
-App_C_Files := App.c ra-tls-server.c sgxsdk-ra-attester_u.c ias-ra.c
-App_Include_Paths := $(Wolfssl_Include_Paths) -I$(SGX_SDK)/include -I$(SGX_RA_TLS_ROOT) -I$(INCDIR) -I$(shell readlink -f .)
+App_C_Files := App.c ra-tls-server.c sgxsdk-ra-attester_u.c
+
+ifndef ECDSA
+App_C_Files += ias-ra.c
```

Comment:
Plz add an indent.

Reply:
So why removing ias-ra.c from this commit if you still support EPID?

Reply:
just for isolating epid and ecdsa. If just compiling EPID, then the file will be compiled, otherwise it will be ignored.